web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1264,10 +1264,10 @@ export const formatters: Formatters = {
   [eventCodes.DESKTOP_SESSION_STARTED]: {
     type: 'windows.desktop.session.start',
     desc: 'Windows Desktop Session Started',
-    format: ({ user, windows_domain, desktop_name, windows_user }) => {
-      let message = `User [${user}] has connected to Windows desktop [${windows_user}@${desktop_name}]`;
+    format: ({ user, windows_domain, desktop_name, sid, windows_user }) => {
+      let message = `User [${user}] started session ${sid} on Windows desktop [${windows_user}@${desktop_name}]`;
```

Comment:
Would it make sense to wrap sid in square brackets so it matches how other identifiers are shown in this audit event?

Reply:
No, because Kenny and I are working on removing these awful square brackets :-)

Reply:
Haha, makes sense.